src/modules/dashboard/components/ColonyActions/ColonyActions.tsx

```diff
@@ -125,8 +128,29 @@ const ColonyActions = ({
     },
   });
 
+  const { data: extensions } = useColonyExtensionsQuery({
+    variables: { address: colonyAddress },
+  });
+
+  /*
+   * @NOTE Prettier is being stupid
+   */
```

Comment:
I agree. I think we need to extend the `max-len` to 120 chars

But that needs to be done in a separate PR since that change will trigger `eslint` to refactor the whole code :)